alvr/common/src/logging.rs

```diff
@@ -128,11 +128,12 @@ pub fn show_e_blocking<E: Display>(e: E) {
 }
 
 pub fn show_err<T, E: Display>(res: Result<T, E>) -> Option<T> {
-    res.map_err(|e| show_e_block(e, false)).ok()
+    res.map_err(|e| show_e_block(format!("{:#}", e), false))
+        .ok()
 }
 
 pub fn show_err_blocking<T, E: Display>(res: Result<T, E>) -> Option<T> {
-    res.map_err(|e| show_e_block(e, true)).ok()
+    res.map_err(|e| show_e_block(format!("{:#}", e), true)).ok()
 }
```

Comment:
Why are you changing this code? I'm surprised it even compiles. `{:#}` is valid for `E: Debug` but the type bound is not specified. In practice, try not to depend on Debug, rather depend on Display.

Reply:
Ah, an explanation of what happened here:
While testing the "no unwrap" changes, I found that I wasn't getting the "inner error" (i.e. the `ErrorUnknown` from the media error), just the context. Now, to me, this is very bad (because it makes it harder to debug codec error reports from users/etc).
However, I couldn't get the error details out of it without either duplicating the inner error details in the context or going through this. I'll change things to duplicate the inner error details in the context.

Reply:
Remember you can get help from anyhow to format the error

Reply:
Turns out that wasn't necessary. Not sure what I missed the first time; doesn't matter in any case.
https://github.com/alvr-org/ALVR/pull/1933/commits/d3645feee654da71fa3ec9f7b49bb6e0c9e74845 undoes this.